app/assets/javascripts/modules/accordion-with-descriptions.js

```diff
@@ -45,6 +47,24 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       bindToggleForSubsections();
       bindToggleOpenCloseAllButton();
 
+      // When navigating back in browser history to the accordion, the browser will try to be "clever" and return
```

Comment:
When the user hits back, shouldn't the same accordion parts be open – wouldn't that put the user in the right place?